GhostProjectBackend/settings.py

```diff
@@ -38,6 +47,10 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'rest_framework',
+    'rest_framework_simplejwt',
```

Comment:
We should also make a requirements.txt file so that we can install dependencies at once. This can be done using pip freeze > requirements.txt  and then we can install them using pip install -r requirements.txt. It is common to name this file "requirements", but you can name it however you want. We also need README.md file, which will contain all the necessary steps which are needed to run this project, for example.. installing MySQL, python 3, pip, etc.